internal/usage.go

```diff
@@ -38,9 +38,10 @@ The commands & flags are:
                                  'processors', 'aggregators' and 'inputs'
   --sample-config                print out full sample configuration
   --once                         enable once mode: gather metrics once, write them, and exit
-  --test                         enable test mode: gather metrics once and print them
-  --test-wait                    wait up to this many seconds for service
-                                 inputs to complete in test or once mode
+  --test                         enable test mode: gather metrics once and print them.
+                                 NO OUTPUTS ARE WRITTEN!
```

Comment:
Same as above "executed" or "used", and I don't think we need to caps it all

Reply:
Tried to make it "bold". Will switch to "executed", even though I think "written" is more clear.

Reply:
I thought saying outputs were not executed is more explicit. Saying outputs are not written, could lead someone to think outputs are still run.